server/src/main/java/org/candlepin/dto/api/v1/UserDTO.java

```diff
@@ -0,0 +1,249 @@
+    protected String password;
+    @ApiModelProperty(required = false, example = "true")
+    protected Boolean superAdmin;
+
+    /**
+     * Initializes a new UserDTO instance with null values.
+     */
+    public UserDTO() {
+        // Intentionally left empty
+    }
+
+    /**
+     * Initializes a new UserDTO instance which is a shallow copy of the provided
+     * source entity.
+     *
+     * @param source
+     *  The source entity to copy
+     */
+    public UserDTO(UserDTO source) {
+        super(source);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public UserDTO populate(UserDTO source) {
+        super.populate(source);
+
+        this.setId(source.getId());
+        this.setUsername(source.getUsername());
+        this.setPassword(source.getPassword());
+        this.setSuperAdmin(source.isSuperAdmin());
+
+        return this;
+    }
+
+    /**
+     * Fetches this user's ID. If the ID has not been set, this method returns null.
+     *
+     * @return
+     *  This user's ID, or null if the ID has not been set
+     */
+    public String getId() {
+        return this.id;
+    }
+
+    /**
+     * Sets or clears the ID for this user.
+     *
+     * @param id
+     *  The ID to set for this user, or null to clear any previously set ID
+     *
+     * @return
+     *  a reference to this DTO
+     */
+    public UserDTO setId(String id) {
+        this.id = id;
+        return this;
+    }
+
+    /**
+     * Fetches this user's username. If the username has not been set, this method returns null.
+     *
+     * @return
+     *  This user's username, or null if the username has not been set
+     */
+    public String getUsername() {
+        return this.username;
+    }
+
+    /**
+     * Sets or clears the username for this user.
+     *
+     * @param username
+     *  The username to set for this user, or null to clear any previously set username
+     *
+     * @return
+     *  a reference to this DTO
+     */
+    public UserDTO setUsername(String username) {
+        this.username = username;
+        return this;
+    }
+
+    /**
+     * Fetches the password set for this user. If the password hasn't been set, this method returns
+     * null.
+     * <p></p>
+     * <strong>
+     *  Note that for obvious reasons, this be serialized for output under any sane circumstances.
+     * </strong>
+     *
+     * @return
+     *  The password set for this user, or null if the password hasn't been set.
+     */
+    @JsonIgnore
+    public String getPassword() {
+        return this.password;
+    }
+
+    /**
+     * Sets or clears the plain-text password to use for this user.
+     *
+     * @param password
+     *  The password, in plain text, to set for this user, or null to clear any previously set
+     *  password
+     *
+     * @return
+     *  a reference to this DTO
+     */
+    public UserDTO setPassword(String password) {
```

Comment:
I believe you need to add a `@JsonProperty` on this setter explicitly, because the `@JsonIgnore` annotation on the getter marks this whole field (field, setter, getter) as ignored.

Reply:
Fixed